windows/src/test/unit-tests/kmcomp/test_casedkeys.kmn

```diff
@@ -0,0 +1,39 @@
+c Tests cased keys rule expansions
+store(&NAME) 'Test &CasedKeys'
+begin Unicode > use(main)
+
+group(main) using keys
+
+store(&CasedKeys) [K_A] [K_E] [K_I] [K_O] [K_U] [K_1] .. [K_4]
```

Comment:
Are [VK_NUMPAD0] .. [VK_NUMPAD9] valid to use in `&CasedKeys` store?

Reply:
Sure, you could do that (`[K_NP0] .. [K_NP9]`)